modular_skyrat/master_files/code/modules/client/preferences/middleware/food.dm

```diff
@@ -0,0 +1,121 @@
+/// To make this system not a massive meta-pick for gamers, while still allowing plenty of room for unique combinations.
+#define MINIMUM_REQUIRED_TOXICS 1
+#define MINIMUM_REQUIRED_DISLIKES 2
+#define MAXIMUM_LIKES 1
+
+GLOBAL_DATUM_INIT(food_prefs_menu, /datum/food_prefs_menu, new)
+
+GLOBAL_LIST_INIT(food_flag_to_bitflag, list(
+	"MEAT" = MEAT,
+	"VEGETABLES" = VEGETABLES,
+	"RAW" = RAW,
+	"JUNKFOOD" = JUNKFOOD,
+	"GRAIN" = GRAIN,
+	"FRUIT" = FRUIT,
+	"DAIRY" = DAIRY,
+	"FRIED" = FRIED,
+	"ALCOHOL" = ALCOHOL,
+	"SUGAR" = SUGAR,
+	"GROSS" = GROSS,
+	"TOXIC" = TOXIC,
```

Comment:
GROSS, TOXIC, JUNKFOOD, GORE, and RAW need to be unselectable in the system, this isn't for fluff, these are the mechanics we use to encourage people to actually eat chef food and not just chow down on raw monkey meat slabs and chips all day long.

Reply:
Other traits (the dog one) should mayhaps get gore as a plus in exchange for chocolate becoming death.

Out of scope. Goof is correct, one of the many mottos of SS13 coding is; if they can powergame, assume they will.

Reply:
Already been brought up, already will be "balanced" around a believable tastes system I'm drawing up to make GROSS, TOXIC, JUNKFOOD, GORE, and RAW all far less appealing options when weighed up with the potential benefits of other foods.

Reply:
@ me when this is implemented, I'd like to see this.

Reply:
> deviant tastes inversing your food likes along with costing some points

Deviant tastes is neutral actually

Reply:
Also, for the quirk, it's made redundant by this system, so I can quite easily just disable the quirk without issue.

Reply:
What if you want to use the quirk and play the game with species default tastes

Reply:
Having had a closer look at species code, I'm gonna bump some values around to allow for being able to recreate most species' likes.

The quirks aren't making a return, though. They're more or less completely unnecessary with this new system.

You can quite easily recreate deviant tastes with my current values (not pushed yet) anyways.